Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -133,6 +133,33 @@ class MessageDTO: NSManagedObject {
         ])
     }
     
+    /// Returns predicate with thread messages that should be shown in the thread.
+    static func threadRepliesPredicate(for messageId: MessageId) -> NSCompoundPredicate {
```

Comment:
Since this logic is very similar not channel messages, should we find a way how to share this logic to prevent similar inconsistencies in the future?